activemodel/lib/active_model/validations/exclusion.rb

```diff
@@ -29,7 +29,9 @@ module HelperMethods
       # Configuration options:
       # * <tt>:in</tt> - An enumerable object of items that the value shouldn't
       #   be part of. This can be supplied as a proc, lambda or symbol which returns an
-      #   enumerable. If the enumerable is a range the test is performed with
+      #   enumerable. If the enumerable is a numerical, time or datetime range the test
+      #   is performed with <tt>Range#cover?</tt>, otherwise with <tt>include?</tt>. When
```

Comment:
I actually took the original comment from a [comment in inclusion validator](https://github.com/rails/rails/blob/master/activemodel/lib/active_model/validations/inclusion.rb#L32) which was already made with `<tt>` and put it here since it is also relevant here. 

I don't personally have a strong preference on this, but I think consistency is important. If we should use `++` for all simple, one word phrases, then there are a very large number of places in the source that would also need to be changed. 

The way the rails source currently looks is that most of the places where fixed font is needed in the middle of some text use `<tt>` and `++` is used in lists (for example where common options are just listed one after the other). Because of that, using `<tt>` here seemed more in line with the rest of the codebase.